hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/impl/TestContainerDeletionChoosingPolicy.java

```diff
@@ -109,27 +111,37 @@ public void testRandomChoosingPolicy() throws IOException {
     }
     blockDeletingService = getBlockDeletingService();
 
+    int blockLimitPerInterval = 5;
     ContainerDeletionChoosingPolicy deletionPolicy =
         new RandomContainerDeletionChoosingPolicy();
-    List<ContainerData> result0 =
-        blockDeletingService.chooseContainerForBlockDeletion(5, deletionPolicy);
-    Assert.assertEquals(5, result0.size());
+    List<ContainerBlockInfo> result0 = blockDeletingService
+        .chooseContainerForBlockDeletion(blockLimitPerInterval, deletionPolicy);
+
+    if (result0.size() > 0) {
```

Comment:
We should remove the if conditions in this test related to result0, result1 and result2 size.

Reply:
Removed the 'if' condition.